hack/e2e.sh

```diff
@@ -108,6 +109,32 @@ ssh-keygen -t ed25519 -f "${VSPHERE_SSH_PRIVATE_KEY}" -N ""
 export VSPHERE_SSH_AUTHORIZED_KEY
 VSPHERE_SSH_AUTHORIZED_KEY="$(cat "${VSPHERE_SSH_PRIVATE_KEY}.pub")"
 
+# Ensure vSphere is reachable
+function wait_for_vsphere_reachable() {
+  local n=0
+  until [ $n -ge 300 ]; do
+    curl -s -v "https://${VSPHERE_SERVER}/sdk" --connect-timeout 2 -k && RET=$? || RET=$?
+    if [[ "$RET" -eq 0 ]]; then
+      break
+    fi
+    n=$((n + 1))
+    echo "Failed to reach https://${VSPHERE_SERVER}/sdk. Retrying in 1s ($n/30)"
```

Comment:
Perhaps a typo here `($n/30)`, should be `($n/300)`, and this could be fixed next time.

Reply:
Yup